README.md

```diff
@@ -263,6 +263,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [graphql-crystal](https://github.com/ziprandom/graphql-crystal) - [Graphql](http://graphql.org) implementation
  * [kave](https://github.com/jwoertink/kave) - Kemal API Version Extension
  * [kemal-session](https://github.com/kemalcr/kemal-session) - Session handler for Kemal
+ * [mochi](https://github.com/gitlato/mochi) - Authentication shard inspired by Devise. Includes Authenticable, Confirmable, Invitable & more.
```

Comment:
```suggestion
 * [mochi](https://github.com/gitlato/mochi) - Authentication shard inspired by Devise supporting: Authenticable, Confirmable, Invitable & more.
```

Makes it a single sentence like majority of other shards, and removed the ending period for consistency.